### ISSUE TITLE
I have some error when i launch

### ISSUE THREAD

**Issue by ddiawara** [mschuchard/packer-plugin-testinfra#5]
My configuration 

````
  provisioner "testinfra" {
    processes   = 2
    #local = true
    pytest_path = "/Users/mission/.mvenv/bin/pytest"
    sudo        = false
    test_files  = ["/Users/mission/client/ZenBuild/PACKER/tests/test_simple.py"]
    #test_files  = ["${path.root}/tests/test_simple.py"]
    #pytest_path = "/usr/local/bin/py.test"
  }
````

````
==> d3n-packer.amazon-ebs.amzn2-0day: Testinfra errored internally during execution:
==> d3n-packer.amazon-ebs.amzn2-0day: ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
==> d3n-packer.amazon-ebs.amzn2-0day: pytest: error: unrecognized arguments: -n 2
==> d3n-packer.amazon-ebs.amzn2-0day:   inifile: None
==> d3n-packer.amazon-ebs.amzn2-0day:   rootdir: /Users/mission/client/ZenBuild/PACKER
==> d3n-packer.amazon-ebs.amzn2-0day: 
==> d3n-packer.amazon-ebs.amzn2-0day:
==> d3n-packer.amazon-ebs.amzn2-0day: Testinfra returned non-zero exit status: %s
==> d3n-packer.amazon-ebs.amzn2-0day: the pytest Testinfra execution failed
==> d3n-packer.amazon-ebs.amzn2-0day: exit status 4
==> d3n-packer.amazon-ebs.amzn2-0day: Step "StepProvision" failed
==> d3n-packer.amazon-ebs.amzn2-0day: [c] Clean up and exit, [a] abort without cleanup, or [r] retry step (build may fail even if retry succeeds)? 
````

**Comment by mschuchard** [comment]
Does your version of pytest support the usage argument for number of processes?

**Comment by ddiawara** [comment]
pytest             8.0.0
pytest-testinfra   10.1.0

**Comment by mschuchard** [comment]
I have discovered this is a simple documentation issue https://testinfra.readthedocs.io/en/latest/invocation.html#parallel-execution. I will add code to handle specifying the `processes` parameter according to the installation of the plugin, and update the usage documentation.

**Comment by mschuchard** [comment]
Fixed in d024ed8. Unit test for this is behaving oddly, but acceptance testing verified expected behavior.
